air/src/execution_step/instructions/call/call_result_setter.rs

```diff
@@ -52,11 +58,17 @@ pub(crate) fn populate_context_from_peer_service_result<'i>(
                 stream.position,
             );
             let generation = exec_ctx.streams.add_stream_value(value_descriptor)?;
-            Ok(CallResult::executed_stream(cid, generation))
+            Ok(CallResult::executed_stream(service_result_agg_cid, generation))
+        }
+        CallOutputValue::None => {
+            let mut dummy_cid_state = ExecutionCidState::new();
+            let service_result_agg_cid = dummy_cid_state
+                .insert_value(executed_result.result, tetraplet, argument_hash)
+                .map_err(UncatchableError::from)?;
+
+            // TODO: Might we add only value's CID for the `Unused`?
```

Comment:
I think yes because tetraplets and hash arguments can't be used in this case; moreover, creating a dummy `ExecutionCidState` takes time and consumes memory for hashmaps.